packages/bundler-plugin-core/package.json

```diff
@@ -34,6 +34,7 @@
     "lint": "eslint ./src ./test"
   },
   "dependencies": {
+    "@sentry/cli": "^1.74.6",
```

Comment:
Unfortunately, we'll still have to go with v1 as long as we don't drop Node 8 and 10 support